### PR TITLE
Add inpututil guide comment for Is{Key,MouseButton,Gamepad}Pressed and TouchIDs

### LINKED ISSUE #1344: ebiten: Add comments about inpututil at IsKeyPressed and so on
Feedback from @raa0121

### PR DESCRIPTION
**Pull request by raa0121**
close #1344

### CODE REVIEW

File: input.go [hunk 1]
```diff
@@ -174,6 +180,8 @@ func IsGamepadButtonPressed(id int, button GamepadButton) bool {
 }
 
 // TouchIDs returns the current touch states.
+// If you want to know whether the TouchIDs started being pressed in the current frame,
```
Comment (hajimehoshi) [inline input.go:183]: Add a blank line above.

s/TouchIDs/a touch/

File: input.go [hunk 2]
```diff
@@ -37,6 +37,8 @@ func InputChars() []rune {
 }
 
 // IsKeyPressed returns a boolean indicating whether key is pressed.
+// If you want to know whether the key started being pressed in the current frame,
```
Comment (hajimehoshi) [inline input.go:40]: Add a blank line above.

File: input.go [hunk 3]
```diff
@@ -91,6 +93,8 @@ func Wheel() (xoff, yoff float64) {
 }
 
 // IsMouseButtonPressed returns a boolean indicating whether mouseButton is pressed.
+// If you want to know whether the mouseButton started being pressed in the current frame,
```
Comment (hajimehoshi) [inline input.go:96]: ditto

File: input.go [hunk 4]
```diff
@@ -162,6 +166,8 @@ func GamepadButtonNum(id int) int {
 }
 
 // IsGamepadButtonPressed returns the boolean indicating the given button of the gamepad (id) is pressed or not.
+// If you want to know whether the given button of gamepad (id) started being pressed in the current frame,
```
Comment (hajimehoshi) [inline input.go:169]: ditto

**Review by hajimehoshi — COMMENTED** [review]
Thank you!

File: input.go (hunk 1 above)
Comment (raa0121) [reply input.go:183]: Is it s/the TouchIDs/a touch/  ?

Comment (hajimehoshi) [reply input.go:183]: Ah yes, right.

Comment (raa0121) [reply input.go]: fix on af604ff18148f605449be91ecb59a99e5bb08e71

File: input.go (hunk 2 above)
Comment (raa0121) [reply input.go:41]: fix on fix on af604ff18148f605449be91ecb59a99e5bb08e71

File: input.go (hunk 3 above)
Comment (raa0121) [reply input.go:98]: fix on fix on af604ff18148f605449be91ecb59a99e5bb08e71

File: input.go (hunk 4 above)
Comment (raa0121) [reply input.go:172]: fix on fix on af604ff18148f605449be91ecb59a99e5bb08e71

**Review by hajimehoshi — APPROVED** [review]
LGTM, thanks!

**Pull request merged**